app/experimenter/nimbus-ui/README.md

```diff
@@ -236,15 +236,29 @@ While an experiment is in `draft` status all fields are technically allowed to b
 
 No additional action is required; as long as you correctly set the form group's `controlId` and the form error's `name` these review-readiness messages will automatically appear as needed when a user attempts to go into review with incomplete fields.
 
-## Access Visualization Data Locally
+## Results Page and Visualization Data
 
 The Results page renders experiment analysis data and can only be accessed when an experiment is complete, the analysis feature flag is on, and the analysis data returned from the visualization API endpoint is available.
 
+### Locally Access Visualization Data for Experiments in Prod
+
+At the time of writing, locally created complete experiments don't return data from the visualization endpoint. However, it is possible to view a locally created experiment that mirrors an experiment in production to test the Results page locally with production visualization data.
+
 To see the Results page and data locally:
 
 1. Ensure your [Google Credentials](https://github.com/mozilla/experimenter#google-credentials) are configured
 2. Set `FEATURE_ANALYSIS=true` in your local `.env` file
-3. Create a Nimbus Experiment at `localhost/admin` with experiment slug "testing-nimbus" status "complete," a branch/branch slug named "control", and another branch/branch slug named "treatment". Alternately, change an existing, complete Nimbus experiment with branches in place to reflect the experiment slug `testing-nimbus`.
+3. Find a completed experiment in production that has analysis data ready that you'd like to test your changes against
+4. At `localhost/admin`, add a new Nimbus Experiment with the same experiment slug as the production experiment you'd like to test against. Change the status to "complete," and add every branch (name and slug) to match the production experiment. Also, make sure "reference branch" is set to the reference branch for that production experiment.
```

Comment:
I think you actually only need to add the reference branch's name and slug and the rest is picked up on its own. This is handy if you want to try out an experiment locally that has a lot of branches so it's less time to have to input all of them.